tests/integration/test_charm.py

```diff
@@ -107,27 +108,23 @@ async def test_setup_discourse(
 
     # Get the form info
     assert parsed_registration.body
-    utf8 = parsed_registration.body.find("input", attrs={"name": "utf8"}).get("value")  # type: ignore
-    authenticity_token = parsed_registration.body.find("input", attrs={"name": "authenticity_token"}).get("value")  # type: ignore
-    form_fields = {
-        "utf8": utf8,
-        "authenticity_token": authenticity_token,
-        "username": "admin",
-        "email": app_config["developer_emails"],
-        "password": "MyLovelySecurePassword2022!",
-    }
-
-    form_headers = {
-        "Host": f"{app_config['external_hostname']}",
-        "Content-Type": "application/x-www-form-urlencoded",
-    }
-
     logger.info("Submitting registration form")
 
     response = session.post(
         f"{discourse_url}/finish-installation/register",
-        headers=form_headers,
-        data=urlencode(form_fields),
+        headers={
+            "Host": f"{app_config['external_hostname']}",
+            "Content-Type": "application/x-www-form-urlencoded",
+        },
+        data=urlencode(
+            {
+                "utf8": parsed_registration.body.find("input", attrs={"name": "utf8"}).get("value"),  # type: ignore # pylint: disable=line-too-long
```

Comment:
I think you can resolve this line to long as well using something like:

```Python
parsed_registration.body.find(
    "input", attrs={"name": "utf8"}
).get("value")  # type: ignore
```

Not sure where the type: ignore needs to go, just guessing

Reply:
This comment also applies to other disabling of line too long in this PR

Reply:
Awesome, I tried something similar yesterday, but black reformatted it, however with your example it works! Thanks a lot!